packages/dashboard-core-plugins/src/linker/Linker.tsx

```diff
@@ -666,7 +702,7 @@ export class Linker extends Component<LinkerProps, LinkerState> {
         ? 'Click a column source, then click a column target to create a filter link. Remove a filter link by clicking again to erase. Click done when finished.'
         : 'Create a link between the source column button and a table column by clicking on one, then the other. Remove the link by clicking it directly. Click done when finished.';
 
-    return (
+    return isLinkOverlayShown ? (
```

Comment:
Shouldn't need a ternary here, just keep it in the `CSSTransition` `in` attribute - then there will be an appropriate fade animation when you start dragging, and another fade animation when you drop.

Reply:
> After trying it, I agree. You are right we should hide it then re-show it instead.
> 
> Could make it unmount the children, or display none on a parent, but it should be with zero-transition as picking up the component is instant. Looks still a bit weird to have it fade out in the wrong spot as they fade out.

I think we are trying to avoid the fade animation. Is there a better way to get the overlay to disappear as quickly as possible?